rust/operator-binary/src/env_vars.rs

```diff
@@ -575,37 +575,26 @@ fn execution_server_env_vars(airflow: &v1alpha1::AirflowCluster) -> BTreeMap<Str
         // The execution API server URL can be any webserver (if there
         // are multiple ones). Parse the list of webservers in a deterministic
         // way by iterating over a BTree map rather than the HashMap.
-        if let Some(webserver_role) = airflow.spec.webservers.as_ref() {
-            if let Some(rolegroup) = webserver_role
-                .role_groups
-                .iter()
-                .collect::<BTreeMap<_, _>>()
-                .first_entry()
-            {
-                let webserver = format!(
-                    "{name}-webserver-{rolegroup}",
-                    name = name,
-                    rolegroup = rolegroup.key()
-                );
-                tracing::debug!("Webserver set [{webserver}]");
-                // These settings are new in 3.x and will have no affect with earlier versions.
-                env.insert(
-                    "AIRFLOW__CORE__EXECUTION_API_SERVER_URL".into(),
-                    EnvVar {
-                        name: "AIRFLOW__CORE__EXECUTION_API_SERVER_URL".into(),
-                        value: Some(format!("http://{webserver}:8080/execution/")),
-                        ..Default::default()
-                    },
-                );
-                env.insert(
-                    "AIRFLOW__CORE__BASE_URL".into(),
-                    EnvVar {
-                        name: "AIRFLOW__CORE__BASE_URL".into(),
-                        value: Some(format!("http://{webserver}:8080/")),
-                        ..Default::default()
-                    },
-                );
-            }
+        if airflow.spec.webservers.as_ref().is_some() {
+            let webserver = role_service_name(name, "webserver");
```

Comment:
Can we not hardcode the role?

Reply:
[1bf2615](https://github.com/stackabletech/airflow-operator/pull/645/commits/1bf26155f99cee977b25de8efe7b7098830fe48f)